docs/reference/ilm/apis/explain.asciidoc

```diff
@@ -111,30 +111,33 @@ that the index is managed and in the `new` phase:
   "indices": {
     "my-index-000001": {
       "index": "my-index-000001",
-      "managed": true, <1>
-      "policy": "my_policy", <2>
-      "lifecycle_date_millis": 1538475653281, <3>
-      "age": "15s", <4>
+      "index_creation_date_millis": 1538475653281,
+      "index_age": "15s",                       <1>
+      "managed": true,                          <2>
+      "policy": "my_policy",                    <3>
+      "lifecycle_date_millis": 1538475653281,   <4>
+      "age": "15s",                             <5>
       "phase": "new",
-      "phase_time_millis": 1538475653317, <5>
+      "phase_time_millis": 1538475653317,       <6>
       "action": "complete",
-      "action_time_millis": 1538475653317, <6>
+      "action_time_millis": 1538475653317,      <7>
       "step": "complete",
-      "step_time_millis": 1538475653317 <7>
+      "step_time_millis": 1538475653317         <8>
     }
   }
 }
 --------------------------------------------------
 // TESTRESPONSE[skip:no way to know if we will get this response immediately]
 
-<1> Shows if the index is being managed by {ilm-init}. If the index is not managed by
+<1> The age of the index
```

Comment:
shall we document that this is used by the rollover `max_age` math?

Reply:
Yes, that is a good idea, I will do that.